README.md

```diff
@@ -1,42 +1,36 @@
 # BraveButtons [![Build Status](https://app.travis-ci.com/bravetechnologycoop/BraveButtons.svg?branch=main)](https://app.travis-ci.com/bravetechnologycoop/BraveButtons)
 
-# How to release a new production version & update the entire Button system
+# Production Deployment
 
-The following instructions apply if you are deploying a new version of the code, optionally including
-updates to the config repo and/or changes to the config repo schema. If you are only deploying changes
-to the config repo that do not change the schema (eg. adding new Buttons)
-then it is not necessary to create a new numbered version. In this case you can skip step 1 and skip
-making a new tag during step 2. This is essentially redeploying an older version with new config.
+## 1. Update git to reflect the new release
 
 1. check the deployment ClickUp Task for any comments or SubTasks that could affect these deployment steps.
 
+1. send a message to the `#buttons-aa-general` Slack channel letting everyone know that you are doing a deployment and to expect some downtime.
+
 1. on your local machine, in the `BraveButtons` repository:
 
    1. pull the latest code ready for release: `git checkout main && git pull origin main`
 
    1. decide on an appropriate version number for the new version
 
-   1. update CHANGELOG.md by moving everything in `Unreleased` to a section for the new version
+   1. update CHANGELOG.md by moving everything in `Unreleased` to a section for the new version. Remember to scroll down to the bottom to update the links!
 
-   1. make a new commit directly on `main` which only updates the changelog
+   1. make a new commit directly on `main` which only updates the CHANGELOG
 
    1. tag the new commit - for example, if the version number is v1.0.0, use `git tag v1.0.0`
 
    1. push the new version to GitHub: `git push origin main --tags`
 
    1. update the `production` branch: `git checkout production && git merge main && git push origin production`
 
-1. on your local machine, in the `buttons-config` repository:
-
-   1. login to AWS SSO: `aws sso login --profile brave-management`
-
-   1. ensure you have the latest version of the code: `git pull origin main`
+## 2. Update the Environment Variables (if necessary)
```

Comment:
I am also considering adding the following step:

If you need to install AWS CLI on your machine, please follow the instructions provided in this link: [AWS CLI Installation Guide](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html).

If necessary, follow these steps:

-  In your root directory, create a `.aws` folder.

- Inside the `.aws` folder, use the `nano` text editor to create or edit the `config` file. Paste the `[profile-brave-management]` key configuration into this file.

- If you encounter an issue where running `aws --version` results in `-bash: [route]: no such file or directory`, follow these additional steps:

   - Open the `.bashrc` file using the `nano` text editor.
   - Scroll to the bottom of the file and ensure that the path for your scripting is correctly defined. For example: `export PATH="$PATH:$HOME/.rvn/bin:[route]"`.
   - Save the changes.

-  After saving the `.bashrc` file, run `source .bashrc` to apply the changes.

-  Finally, run `aws --version` again to confirm that everything is functioning correctly.

These steps should help you set up the necessary AWS configurations and ensure that the AWS CLI operates without issues.

Reply:
Hi Ahmad. The environment variables for Buttons live in the `.env` file on the `chatbot.brave.coop` server (or `chatbot-dev.brave.coop in the case of the Development environment).

We don't use AWS for Buttons. So I won't be adding your suggestion here. However, please take a look at this PR about updates to the Sensors README and see if there is a good place to add your suggestion there: https://github.com/bravetechnologycoop/BraveSensor/pull/227